cocos/renderer/gfx-gles3/GLES3Context.cpp

```diff
@@ -137,18 +137,106 @@ bool GLES3Context::initialize(const ContextInfo &info) {
         _colorFmt = Format::RGBA8;
         _depthStencilFmt = Format::D24S8;
 
-        const EGLint attribs[] = {
+        bool msaaEnabled = _device->hasFeature(Feature::MSAA);
+        EGLint redSize{8}, greenSize{8}, blueSize{8}, alphaSize{8}, depthSize{24}, stencilSize{8}, sampleBufferSize{msaaEnabled ? EGL_DONT_CARE : 0}, sampleSize{msaaEnabled ? EGL_DONT_CARE : 0};
+
+        // TODO: there are more options if not limit rgba to 8.
```

Comment:
If you mean float textures, etc. I don't think we need to consider those options. For the purpose of back buffers RGBA8 is perfectly fine.